packages/react-native-hms/android/src/main/java/com/reactnativehmssdk/HMSManager.kt

```diff
@@ -857,6 +857,22 @@ class HMSManager(
     val showAudioButton: Boolean,
   )
 
+  @ReactMethod
+  fun changeIOSPIPVideoTrack(
```

Comment:
we can delete `changeIOSPIPVideoTrack` and `setActiveSpeakerInIOSPIP` methods? @ygit